src/main/java/com/redhat/red/build/finder/Main.java

```diff
@@ -190,11 +195,18 @@ public static void main(String[] args) {
             }
 
             File configFile = configPath.toFile();
-            BuildConfig config;
+            BuildConfig config = null;
 
             if (configFile.exists()) {
                 LOGGER.info("Using configuration file: {}", green(configFile));
-                config = mapper.readValue(configFile, BuildConfig.class);
+
+                try {
+                    config = mapper.readValue(configFile, BuildConfig.class);
+                } catch (IOException e) {
+                    LOGGER.error("Error reading configuration file: {}", boldRed(e.getMessage()));
+                    LOGGER.debug("Error", e);
+                    System.exit(1);
```

Comment:
Should we fail here or just continue with the configuration defaults?

Reply:
I vote for fail ; a person might want to fix a corrupted file but continuing  (even with a warning ) doesn't really give them a chance.